specifyweb/frontend/js_src/lib/components/FormEditor/createView.ts

```diff
@@ -263,38 +263,63 @@ function createViewFromTemplate(
   const originalNames = filterArray(
     view.altViews.altViews.map(({ viewDef }) => viewDef)
   );
+
   const nameMapper = Object.fromEntries(
     originalNames.map((name) => [name, getUniqueDefinitionName(name, viewSets)])
   );
-  return {
-    ...viewSets,
-    views: [
-      ...viewSets.views,
-      {
-        ...view,
-        name,
-        altViews: {
-          ...view.altViews,
-          altViews: view.altViews.altViews.map((altView) => ({
-            ...altView,
-            viewDef:
-              typeof altView.viewDef === 'string'
-                ? nameMapper[altView.viewDef] ?? altView.viewDef
-                : altView.viewDef,
-          })),
-        },
-      },
-    ],
-    viewDefs: [
-      ...viewSets.viewDefs,
-      ...viewDefinitions.map((definition) => ({
-        ...definition,
+
+  const uniqueName = getUniqueDefinitionName(name, viewSets);
```

Comment:
read line 18 in this file - the name would already be unique (see this component - https://github.com/specify/specify7/blob/a66e7a8681d2833757ad33dc502d636944be7d09/specifyweb/frontend/js_src/lib/components/FormEditor/Create.tsx#L201#)

feel free to copy-paste that comment onto line 256 if that would be helpful for the future